course/src/isometry.cc

```diff
@@ -0,0 +1,83 @@
+/**
+ * @file Vector.c
+ * @author your name (you@domain.com)
+ * @brief
+ * @version 0.1
+ * @date 2020-02-03
+ *
+ * @copyright Copyright (c) 2020
+ *
+ */
+
+#include <isometry.h>
+
+namespace ekumen {
+namespace math {
+Vector3::Vector3(const double &x, const double &y, const double &z) : x_(x), y_(y), z_(z) {}
+
+Vector3::Vector3() {
```

Comment:
Better [delegate the construction](https://www.learncpp.com/cpp-tutorial/8-6-overlapping-and-delegating-constructors/comment-page-1/):

```
Vector3::Vector3() : Vector3(0.0, 0.0, 0.0) {}
```